lib/logflare/backends/source_sup.ex

```diff
@@ -39,14 +46,19 @@ defmodule Logflare.Backends.SourceSup do
         {project_id, dataset_id}
       end
 
-    default_backend =
-      Backend.child_spec(source, %Backend{
-        type: :bigquery,
-        config: %{
-          project_id: project_id,
-          dataset_id: dataset_id
-        }
-      })
+    specs =
+      ([
+         %Backend{
+           type: :bigquery,
+           config: %{
+             project_id: project_id,
+             dataset_id: dataset_id
+           }
+         }
+         | ingest_backends
+       ] ++ rules_backends)
```

Comment:
I find

```suggestion
      [
         %Backend{
           type: :bigquery,
           config: %{
             project_id: project_id,
             dataset_id: dataset_id
           }
         }
         | ingest_backends
       ]
       |> Enum.concat(rules_backends)
```

To be a little bit cleaner?